glean-core/ios/Glean.xcodeproj/xcshareddata/xcschemes/Glean.xcscheme

```diff
@@ -26,6 +26,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      codeCoverageEnabled = "YES"
```

Comment:
What does this do? (and was it an intentional change?)

Reply:
This allows me to see what coverage we have for our tests, it's a handy feature to make @Dexterp37 happy ;)

Reply:
We should see to extract that on CI at some point :)